_posts/fixed-vs-dynamic.md

```diff
@@ -0,0 +1,43 @@
+
+The relatively linear cost growth of GoLink means the subsidy per passenger is often higher than bus or train service, which can create some concerns around the incentive structure for the transit agency. If buses are more cost-effective when they carry more passengers, the transit agency is incentivized to move more people. This leads to better service and more riders — exactly the result you want. However, if you have a system where costs are not decreased by increasing ridership, the transit agency has no incentive to improve.
+
+There are several neighboring cities that only have an on-demand system. One of them is Arlington, where the company Via runs a service similar to GoLink. It’s notorious for long wait times due to lack of drivers, requiring users to continually request rides or give up and take a different ridesharing service. Via is not incentivized to improve this experience because taking more rides does not offset the cost of hiring more drivers, so the service remains suboptimal. On top of these issues, on-demand systems don’t fulfill the other two main goals of transit, reducing traffic and air pollution.
+
+## The Takeaway
+GoLink has been a great addition to DART to serve areas that aren't dense enough for bus service. It serves as a bottom rung on the ladder below buses and trains, which have higher upfront cost but better scalability. If we assume the best intentions, the city officials of Plano and other areas saw the cost reduction of replacing some low-performing bus routes with GoLink in 2019, then made the assumption that these cost reductions could be replicated across the board. If we assume the worst intentions, some of the Plano DART board and city council members have known financial ties to Uber and could be looking to benefit themselves by increasing Uber’s contract with DART. As often is the case, the answer likely lies somewhere in the middle. Regardless of motivation, replacing the entire system with only dynamic transportation would not benefit cities or riders.
+
+### Sources
+- Shelby Williams comment on Reddit, Nov 27, 2024: https://www.reddit.com/r/dart/comments/1gybalr/comment/lz9v4nj/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button
```

Comment:
At the sources, the links are not clickable on the vercel preview, make sure to use the markdown format to add links
```[Text here](url)```

Reply:
lines 41-43